### PR TITLE
ONI-181: Changed canSave method to accomodate for possible 0 value.

### PR DESCRIPTION
**Pull request by wzglinieckisoldevelo**
Ticket: https://openimis.atlassian.net/browse/ONI-181

Changes:
- Added new condition to canSave check.
- Now it allows Payment Grace Period to be equal to 0.
- This change was likely introduced in the generic core component.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const validateProductForm = (values, rules, isProductCodeValid) => {
   const errors = {};
 
   REQUIRED_FIELDS.forEach((field) => {
-    if (!values[field]) {
+    if (!values[field] && values[field] != 0) {
```
Comment (olewandowski1) [inline src/utils.js:22]: Use "!==" instead of "!=".

**Review by olewandowski1 — CHANGES_REQUESTED** [review]
Please refer to the comment. @wzglinieckisoldevelo

File: src/utils.js (hunk 1 above)
Comment (wzglinieckisoldevelo) [reply src/utils.js:22]: I hate this language.

Comment (olewandowski1) [reply src/utils.js:22]: :x:

**Comment by wzglinieckisoldevelo** [comment]
This does not work anyway, after checking saved values. I do not have time to fix it right now, so turning into draft.

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**